src/redux/books/booksSlice.js

```diff
@@ -0,0 +1,17 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  books: [],
+};
+
+const booksSlice = createSlice({
+  name: 'Book',
+  initialState,
+  reducers: {
+    addBook: (state) => state,
+    removeBook: (state) => state,
+  },
```

Comment:
- [ ] It would be nice if you could rename the slice name to books as in `name: "books"` as it's a convention to have our slice names represent the state's name. 👍🏽
- [ ] The addBook reducer is expected to update the state value with the payload value dispatched to it when creating a new book. Kindly work on that 👍🏽
- [ ] As well as the removeBook reducer is expected to remove a book using the filter method to remove a book by the ID passed as a payload. Kindly implement that inside the method 👍🏽